docs/resources/azure_bastion_hosts_resources.md

```diff
@@ -0,0 +1,94 @@
+---
+title: About the azure_bastion_hosts_resource Resource
+platform: azure
+---
+
+# azure_bastion_hosts_resource
+
+Use the `azure_bastion_hosts_resource` InSpec audit resource to test properties related to bastion hots for a resource group or the entire subscription.
+
+## Azure REST API version, endpoint and http client parameters
+
+This resource interacts with api versions supported by the resource provider.
+The `api_version` can be defined as a resource parameter.
+If not provided, the latest version will be used.
+For more information, refer to [`azure_generic_resource`](azure_generic_resource.md).
+
+Unless defined, `azure_cloud` global endpoint, and default values for the http client will be used.
+For more information, refer to the resource pack [README](../../README.md). 
+
+## Availability
+
+### Installation
+
+This resource is available in the [InSpec Azure resource pack](https://github.com/inspec/inspec-azure). 
+For an example `inspec.yml` file and how to set up your Azure credentials, refer to resource pack [README](../../README.md#Service-Principal).
+
+
+Also, refer to [Azure documentation](https://docs.microsoft.com/en-us/rest/api/virtualnetwork/bastion-hosts/list) for  properties available.
+Any attribute in the response may be accessed with the key names separated by dots (`.`).
+## Syntax
+
+An `azure_bastion_hosts_resource` resource block returns all Azure bastion hots, either within a Resource Group (if provided)
+```ruby
+describe azure_bastion_hosts_resource(resource_group: 'my-rg') do
+  
+end
+```
+
+## Properties
+
+|Property       | Description                                                                          | Filter Criteria<superscript>*</superscript> |
+|---------------|--------------------------------------------------------------------------------------|-----------------|
+| name           | A list of the unique resource names.                                                | `name`            |
+| ids            | A list of bastion hosts ids .                                                       | `id`              |
+| tags           | A list of `tag:value` pairs defined on the resources.                               | `tag`             |
+| provisioning_state             | State of BastionHosts creation                                      | `provisioning_state`         |
+| types             |   Types of all the bastion hosts | `type` |
+| properties             |   Types of all the bastion hosts | `property` |
```

Comment:
field as described [here](https://github.com/inspec/inspec-azure/pull/414/files#diff-201aca2a0b9ac95395766f7e16f1328dd1fb4b328bee62705321a4667f51b444R63) should be `properties`